Zad1/src/sql/demo/Main.java

```diff
@@ -0,0 +1,60 @@
+package sql.demo;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) throws Exception{
+        Scanner scanner = new Scanner(System.in);
+        BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+        System.out.println("Введите ширину рамки");
+        int a = scanner.nextInt();
+        System.out.println("Введите длину рамки");
+        int b = scanner.nextInt();
+        System.out.println("Введите строку");
+        String word = reader.readLine();
+
+        //Если длина строки больше ширины рамки выводит "Ошибка"
+        if (b < word.length()) {
+            System.out.println("Ошибка");
+        }
```

Comment:
[ERROR] /Users/geoiva/Desktop/SberJava/students/alena_kochurova/nordic-it-java-q4/Zad1/src/sql/demo/Main.java:23:9: '}' at column 9 should be on the same line as the next part of a multi-block statement (one that directly contains multiple blocks: if/else-if/else, do/while or try/catch/finally). [RightCurly]